### PR TITLE
Revert phasing changes [VS-1144]

### PR DESCRIPTION
**Pull request by mcovarr**
Revert some phasing changes that were unnecessary for AoU and broke our integration tests.

[Integration run](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/ba93baa2-9971-4c90-8ce3-635702a81eb6)

### CODE REVIEW

File: scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl [hunk 1]
```diff
@@ -257,7 +257,7 @@ task ExtractFromSuperpartitionedTables {
                 EXPORT DATA OPTIONS(
                 uri='${avro_prefix}/vets/vet_${str_table_index}/vet_${str_table_index}_*.avro', format='AVRO', compression='SNAPPY') AS
                 SELECT location, v.sample_id, ref, REPLACE(alt,',<NON_REF>','') alt, call_GT as GT, call_AD as AD, call_GQ as GQ, cast(SPLIT(call_pl,',')[OFFSET(0)] as int64) as RGQ,
-                call_PGT as PGT, call_PID as PID, call_PS as PS
+                call_PGT as PGT, call_PS as PS
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:260]: Don't we want to revert call_PGT also?

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:260]: oh geez 🤦

**Review by koncheto-broad — APPROVED** [review]
LGTM

**Review by gbggrant — APPROVED** [review]
Conditional on Integration Tests passing (obv.)

**Pull request merged**